sqldb/migrations_test.go

```diff
@@ -152,3 +158,202 @@ func testInvoiceExpiryMigration(t *testing.T, makeDB makeMigrationTestDB) {
 	require.NoError(t, err)
 	require.Equal(t, expected, invoices)
 }
+
+// getLastMigrationVersion return the version of the last migration in the
+// embedded filesystem.
+func getLastMigrationVersion(t *testing.T) int {
+	t.Helper()
+
+	// Walk through the embedded filesystem
+	entries, err := sqlSchemas.ReadDir("sqlc/migrations")
+	require.NoError(t, err)
+
+	return len(entries)
+}
+
+// TestCustomMigration tests that a custom in-code migrations are executed
+// during the migration process.
+func TestCustomMigration(t *testing.T) {
+	var customMigrationLog []string
+
+	logMigration := func(name string) {
+		customMigrationLog = append(customMigrationLog, name)
+	}
+
+	// Some migrations to use for both the failure and success tests. Note
+	// that the migrations are not in order to test that they are executed
+	// in the correct order.
+	migrations := []MigrationConfig{
```

Comment:
think we should cover the case of having 2 code level migrations applied directly after eachother on same sql level version

Reply:
Should be covered now. PTAL